test/residency/prototypes/test_task014_durable_process_public_seam.py

```
@@ -0,0 +1,1683 @@
+    executable_name = f"task014{suffix}"
+    logical_source = source.relative_to(repo_root).as_posix()
+    logical_output = f"$TMPDIR/{executable_name}"
+    logical_compile_command = compiler_command(
+        compiler, logical_source, logical_output, platform_id
+    )
+    with tempfile.TemporaryDirectory(prefix="residency-task014-") as directory:
+        executable = Path(directory) / executable_name
+        actual_compile_command = compiler_command(
+            compiler_executable, str(source), str(executable), platform_id
+        )
+        subprocess.run(
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)